js/pages/estimation/components/cca-executions.js

```diff
@@ -95,9 +106,31 @@ define([
 		}
 
 		dispose() {
+			super.dispose();
 			PollService.stop(this.pollId);
 		}
 
+		highlightGeneration() {
+			if (this.isSourceListLoaded()) {
+				if (this.notificationSourceId() && this.notificationExecutionId()) {
+					const sourceId = this.notificationSourceId();
+					const executionId = this.notificationExecutionId();
+					const groupIdx = this.executionGroups().findIndex(item => item.sourceId == sourceId);
+					if (groupIdx >= 0) {
+						this.expandedSection() !== groupIdx && this.expandedSection(groupIdx);
+						const row = $(`#${this.notificationRowId()}`);
+						setTimeout(() => {
+							const $row = $(`#${this.notificationRowId()}`);
```

Comment:
What about pagination? I.e. when the row is not on the first page

Reply:
Actually, we don't need to paginate because notification dropdown display last ten jobs. So it will be always the first page